sox.dialog.js

```diff
@@ -89,40 +90,36 @@
         if (featureSettings) {
           const $settingsDiv = $('<div/>', {
             id: 'feature-settings-' + name,
-            'class': 'sox-feature-settings',
+            class: 'sox-feature-settings',
             style: 'display: none; margin-top: 5px;',
           });
 
           const $settingsToggle = sox.sprites.getSvg('wrench', 'Edit this feature\'s settings').click(e => {
             e.preventDefault(); //don't uncheck the checkbox
 
             const $settingsPanel = $('#feature-settings-' + name);
-
-            if ($settingsPanel.is(':visible')) {
-              $settingsPanel.fadeOut();
-            } else {
-              $settingsPanel.fadeIn();
-            }
+            $settingsPanel.is(':visible') ? $settingsPanel.fadeOut() : $settingsPanel.fadeIn();
```

Comment:
```
if ($settingsPanel.is(':visible')) $settingsPanel.fadeOut();
else $settingsPanel.fadeIn();
```

might be better if you are trying to reduce the code and still keep it fairly readable